itou/asp/models.py

```diff
@@ -410,8 +424,10 @@ class Commune(PrettyPrintMixin, AbstractPeriod):
     # What does it mean ? When we autocomplete for an address on the BAN, in 100% of the cases we'll have
     # an INSEE City, thus an ASP Commune and it can be safely used for the employee records.
     city = models.ForeignKey("cities.City", on_delete=models.SET_NULL, verbose_name="ville INSEE", null=True)
+    ignore = models.BooleanField(verbose_name="commune désactivée", default=False)
```

Comment:
On fait pas d'INSERT sur cette table donc ça devrais pas poser de problème mais je me permet un rappel, car on c'est tous fait avoir dans le passé :see_no_evil:, qu'il faut pas oublier `db_default` quand on ajoute un champ NOT NULL si on veux s'éviter des erreurs lors du déploiement :)
```suggestion
    ignore = models.BooleanField(verbose_name="commune désactivée", default=False, db_default=False)
```

Reply:
Merci du rappel !